test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,83 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void one_object_null() {
```

Comment:
Note that our naming convention for test methods is as follows:
`[feature under test/method name]_[scenario]_[expected result]`
e.g. `isAnyNull_emptyList_falseReturned`